ui/src/app/data/interfaces/user-info.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 the original author or authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+export interface UserInfo {
```

Comment:
there is already User interface

Reply:
Nope, this is another interface that has excluded several params as `avatar`, `guid`, `password` ...
In my opinion, this is a better solution to have separate not dependency interfaces from other models